website/pages/docs/[[...page]].jsx

```diff
@@ -1,33 +1,49 @@
 import { productName, productSlug } from 'data/metadata'
-import order from 'data/docs-navigation.js'
 import DocsPage from '@hashicorp/react-docs-page'
+import PluginTierLabel from 'components/plugin-tier-label'
+// Imports below are only used server-side
 import {
   generateStaticPaths,
   generateStaticProps,
-} from '@hashicorp/react-docs-page/server'
+} from 'components/remote-plugin-docs/server'
```

Comment:
On the surface, these `[[...page]]` route files should hopefully look quite familiar.

Reply:
:+1: